### PR TITLE
Some suggestions I have for markup improvements and styling changes

### PR DESCRIPTION
**Pull request by SnailCoil**
### Before:
iphone:
image
ipad portrait:
image
ipad landscape:
image
1920x1080
image

### After:
iphone:
image
ipad portrait:
image
ipad landscape:
image
1920x1080
image

### CODE REVIEW

File: citation_template.html.rsp [hunk 1]
```diff
@@ -1,59 +1,102 @@
...
+                    <% } %>
+                    </ul>
+                <% } %>
+            </section>
+        <% } %>
+        </div>
     </div>
 
+    <!-- Bootstrap Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous"></script>
```
Comment (SnailCoil) [inline citation_template.html.rsp:100]: moved JS to end of file per recommendation: https://getbootstrap.com/docs/5.1/getting-started/introduction/#js

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ map/rsconnect/
 .Rproj.user
 map/map.Rproj
 .DS_Store
+.idea
```
Comment (SnailCoil) [inline .gitignore:8]: 🙈

File: citation_template.html.rsp (hunk 1 above)
Comment (twjordan) [reply citation_template.html.rsp:100]: you rule

File: citation_template.html.rsp [hunk 3]
```diff
@@ -1,59 +1,102 @@
...
+            text-decoration-line: none;
+            color: unset;
+        }
+        .all-caps-link:hover {
+            color: unset;
+        }
+        .attachment + .attachment {
+            margin-top: 3rem;
+        }
+    </style>
```
Comment (SnailCoil) [inline citation_template.html.rsp:41]: I'd strongly recommend extracting these custom styles to their own CSS file, but I didn't take that approach (entirely out of laziness)

**Pull request merged**